rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ProjectSpecTest.java

```diff
@@ -1172,7 +1172,9 @@ public void should_document_get_download_license_info() throws Exception {
                                                 + Arrays.asList("DocxGenerator", "XhtmlGenerator", "TextGenerator")),
                                 parameterWithName("variant").description("All the possible values for variants are "
                                         + Arrays.asList(OutputFormatVariant.values())),
-                                parameterWithName("externalIds").description("The external Ids of the project"))));
+                                parameterWithName("externalIds").description("The external Ids of the project"),
+                                parameterWithName("template").description("The template name")
```

Comment:
It would be good to mention,, that template is for Report and not Disclosure, Since it is a common API for both

Reply:
Mentioned in the decription